bindings/python/tools/python/ostk/astrodynamics/converters.py

```diff
@@ -33,6 +25,9 @@ def coerce_to_datetime(value: Instant | datetime | str) -> datetime:
         return value.get_date_time(Scale.UTC).replace(tzinfo=timezone.utc)
 
     if isinstance(value, str):
+        if value.endswith("Z"):
+            value = value.replace("Z", "+00:00")
+
```

Comment:
Nit: Don't know if this is necessary, tests cases were already covering the "Z" scenario and they seemed fine (see screenshot below):

<img width="731" alt="image" src="https://github.com/user-attachments/assets/e0fd2058-cae5-4143-bf7e-ec7d5a6e520c">

Reply:
So this works if you include the milliseconds, but it doesn't work if you do:
`coerce_to_datetime("2024-01-01T00:00:00Z")`

Reply:
Ah that's because of python3.10 vs python3.11 compatibility. This should make it compatible both ways.